modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs

```diff
@@ -0,0 +1,136 @@
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<div id="docEditDiv" style="display:none">
+    <div class="row-fluid">
+        <div class="span12">
+            <!-- Row -->
+            <form id="addNewDoc" enctype="multipart/form-data" action="apis/upload_documents"
+                  onsubmit="return validateDocs()" method="POST" style="margin-left: 20px;" target="_blank">
+                <div class="container-fluid" style="padding-left:0">
+                    <div id="newDoc">
+                        <div class="row-fluid">
+                            <div class="span6 name-column" style="float:left;width:50%;">
+                                <div class="control-group">
+                                    <label class="control-label" for="docName">Name<span
+                                            class="requiredAstrix"
+                                            style="color: red;">*</span></label>
+
+                                    <div class="controls">
+                                        <input type="text" class="required validInput"
+                                               id="docName" name="docName"
+                                               style="width:100%">
+                                    </div>
+                                </div>
+
+                                <div class="control-group">
+                                    <label class="control-label" for="textarea">Summary</label>
+
+                                    <div class="controls">
+                                        <textarea id="summaryDoc" name="summaryDoc" rows="3"
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7